### ISSUE TITLE
gearmand with mysql queue type - high CPU load

### ISSUE THREAD

**Issue by remizyaka** [gearman/gearmand#156]
Hi, 
Installed latest version of gearmand(1.1.18) on ubuntu 16.
Have 12 different workers but queue is empty most of the time. But for some reason gearman consuming to much CPU resources. 30-100% all the time.
```/usr/sbin/gearmand --listen=127.0.0.1 --pid-file=/run/gearman/server.pid --log-file=/var/log/gearman-job-server/gearman.log --queue-type=MySQL --mysql-host=localhost --mysql-port=3306 --mysql-user=username --mysql-password=password --mysql-db=dbname --mysql-table=gearman_queue --http-port 90210 --hashtable-buckets=1733003```
What can cause such a big CPU load on server without active jobsa and 0 queue? How can I optimise it?

**Comment by remizyaka** [comment]
any thoughts on this?

**Comment by esabol** [comment]
Yeah, it’s not in the docs. Feel free to open another issue regarding that.
